repos/system_upgrade/el7toel8/actors/selinux/selinuxcontentscanner/tests/component_test.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+import pytest
+
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
+from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.reporting import Report
+
+test_modules = [
+    ["400", "mock1"],
+    ["99", "mock1"],
+    ["200", "mock1"],
+    ["400", "mock2"],
+    ["999", "mock3"]
+]
+
+semanage_commands = [
+    ['fcontext', '-t', 'httpd_sys_content_t', '"/web(/.*)?"'],
+    ['fcontext', '-t', 'ganesha_var_run_t', '"/ganesha(/.*)?"'],
+    ['port', '-t', 'http_port_t', '-p', 'udp', '81'],
+    ['permissive', 'abrt_t']
+]
+
+testmoduledir = os.path.join(os.getcwd(), "tests/mock_modules/")
+
+
+def setup_():
+    for priority, module in test_modules:
+        try:
+            semodule = run(["semodule", "-X", priority, "-i", os.path.join(testmoduledir, module + ".cil")])
+        except CalledProcessError as e:
+            api.current_logger().warning("Error installing mock module: %s", e.stderr)
+            api.current_logger().warning("Error installing mock module: %s, %s", str(e.stderr),
+                                         semodule.get("stderr", "fuck"))
```

Comment:
Rather avoid that word in public project :-P
```suggestion
                                         semodule.get("stderr", "o-oh"))
```

Reply:
Sorry about that, I was testing how to properly report the error message (got a little frustrated there) and forgot the code there.